pgmpy/tests/test_models/test_JunctionTree.py

```diff
@@ -1,5 +1,7 @@
 from pgmpy.models import JunctionTree
+from pgmpy.factors import Factor
 from pgmpy.tests import help_functions as hf
+import numpy as np
```

Comment:
@Pratik151 Check PEP8 on how should we group our imports. https://www.python.org/dev/peps/pep-0008/#imports